tutorial/tutorial.md

```diff
@@ -0,0 +1,245 @@
+<!--
+SPDX-Copyright: Free Software Foundation Europe e.V.
+
+SPDX-License-Identifier: CC-BY-SA-4.0
+-->
+
+# Tutorial: How to become REUSE compliant
+
+This tutorial explains the basic methods of how to make a software project REUSE
+compliant. By the end of this document, all your files will clearly have their
+copyright and licensing marked, and you will be able to verify this using a
+linter tool.
+
+Making your project REUSE compliant can be done in three simple steps:
+
+1. Choose and provide licenses
+2. Add copyright and license information to each file
+3. Confirm REUSE compliance
+
+For the purpose of this tutorial, we will assume that the directory of your
+project looks like this:
+
+```
+project/
+├── img/
+│   ├── cat.jpg
+│   └── dog.jpg
+├── src/
+│   └── main.c
+├── .gitignore
+├── Makefile
+└── README.md
+```
+
+If you would like to reproduce the steps in this tutorial on your own
+computer, you can clone the [example
+repository](https://github.com/fsfe/reuse-example). The branch `noncompliant`
+matches the structure above, while the `master` branch is the successful
+result of this repository.
+
+## 1. Choose and provide licenses
+
+The first thing you need to do is to [choose a
+license](https://choosealicense.com/). For this tutorial, we assume that you
+chose the GNU General Public Licence v3.0 or any later version (GPL). More
```

Comment:
This doesn't make a lot of sense in context, because SPDX identifiers haven't been introduced yet:

The first thing you need to do is to [choose a
license](https://choosealicense.com/). For this tutorial, we assume that you
chose the GNU General Public Licence v3.0 or any later version (GPL). More
than simply choosing a license, you need to put the license in your project
directory.

You find your license in the [SPDX License List](https://spdx.org/licenses/).
SPDX is an open standard for communicating license and copyright information.
Each license is uniquely identified by a shortform [SPDX
identifier](https://spdx.org/ids). The SPDX identifier for your chosen license
is GPL-3.0-or-later.